include/rpminspect.h

```diff
@@ -401,12 +401,13 @@ const char *get_before_debuginfo_path(struct rpminspect *ri, const char *binarch
  * IMPORTANT: Do not free the returned string.
  *
  * @param ri The struct rpminspect for the program.
+ * @param file The file we are looking for debuginfo for.
```

Comment:
This could probably be re-worded a bit; maybe something like

"The file that debuginfo should be associated with"

Or something along those lines.